core/services/ocr2/plugins/threshold/decryption_queue.go

```diff
@@ -105,7 +105,7 @@ func (dq *decryptionQueue) getResult(ciphertextId decryptionPlugin.CiphertextId,
 
 	req, ok := dq.completedRequests[string(ciphertextId)]
 	if ok {
-		dq.lggr.Debugf("ciphertextId %s was already decrypted by the DON", ciphertextId)
+		dq.lggr.Debugf("ciphertextId %s was already decrypted by the DON", string(ciphertextId))
```

Comment:
Is ciphertext printable? Shouldn't we convert to hex before logging?
Also Debugw() is preferable.

Reply:
I am already using debugf everywhere else.  Unless you have a strong preference, I think we should keep it.

Also, when looking at the logs, they appear as I would expect with the log messages as they appear in the PR.